onnxruntime/core/session/inference_session.cc

```diff
@@ -1280,6 +1280,9 @@ Status AssignNodesToEpsFromHashesImpl(Graph& graph, const fbs::SessionState& fbs
   for (const auto& node : graph.Nodes()) {
     if (node.GetExecutionProviderType().empty()) {
       auto kernel_hash = utils::GetHashValueFromStaticKernelHashMap(node.OpType(), node.SinceVersion());
+      if (!kernel_hash.has_value()) {
+        kernel_hash = utils::GetInternalNhwcOpHash(node);
+      }
       if (kernel_hash.has_value()) {
```

Comment:
Ugh. We really need to fix this setup of doing the same hash lookups in both inference_session and session_state.

Reply:
Do you know why this wasn't caught by CheckNhwcTransformerIsApplied? That uses and ORT format model and I thought ran in a minimal build.

Reply:
Will clean up the hash lookup and add/update tests in a separate change so we can cherry-pick this one first.